### DISCUSSION TITLE
Use of scripts after the application update is finished

### DISCUSSION THREAD

**Discussion by amitjoy** [shannah/jdeploy#65] [Q&A]
I have noticed the following in the generated `package.json` file -

`"scripts": {"test": "echo \"Error: no test specified\" && exit 1"},` which seems to me that it is supposed to be executed after the installation or update is finished, right? I have a requirement that after the `update` of the application is finished, I would like to delete a folder from the home directory if it exists.

According to our previous discussion in https://github.com/shannah/jdeploy/issues/55, the current working directory is set to home directory, right? I hope the script is also executed in the same directory.

I am also curious to know how it would be handled across different platforms.

Could you kindly enlighten on the scripts as I could not find extensive documentation for it?

**Comment by shannah** [comment]
That is an npm-specific thing.  It isn't used by jdeploy.
